src/main/java/uk/gov/hmcts/cmc/claimstore/courtfinder/CourtFinderApi.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.cmc.claimstore.courtfinder;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import uk.gov.hmcts.cmc.claimstore.courtfinder.models.Court;
+import uk.gov.hmcts.cmc.claimstore.courtfinder.models.CourtDetails;
+
+import java.util.List;
+
+@FeignClient(name = "court-finder-api", url = "https://courttribunalfinder.service.gov.uk")
```

Comment:
can we make this a property e.g. `@FeignClient(name = "idam-api", url = "${idam.api.url}")`